test/msssim.jl

```diff
@@ -0,0 +1,73 @@
+using ImageFiltering
+
+@testset "MS-SSIM" begin
+    @info "test: MS-SSIM"
```

Comment:
Should also add a test to check if `MSSSIM(KernelFactors.gaussian(1.5, 11), (1, ))` outputs the same results to `SSIM()`